pkg/registry/registry.go

```diff
@@ -88,7 +110,7 @@ func (endpoint *RegistryEndpoint) GetTags(img *image.ContainerImage, regClient R
 			} else if endpoint.TagListSort == TagListSortLatestLast {
 				ts = i
 			}
-			imgTag := tag.NewImageTag(tagStr, time.Unix(int64(ts), 0), "")
+			imgTag := tag.NewImageTagWithTagPart(tagStr.original, tagStr.found, time.Unix(int64(ts), 0), "")
```

Comment:
Hm. I think `tagStr` should be renamed to `tagInfo` or something like that in the loop to better reflect the fact that it's not a string anymore. Using `tagStr` was a bad decision in the first place, but now it just becomes confusing :)